src/main/java/com/odysseusinc/arachne/datanode/controller/ExceptionHandlingAdvice.java

```diff
@@ -155,4 +166,28 @@ private String generateErrorToken() {
         return UUID.randomUUID().toString();
     }
 
+    @ExceptionHandler({NoHandlerFoundException.class})
+    public void handleNotFoundError(HttpServletRequest request, HttpServletResponse response) throws Exception {
```

Comment:
This code block is equal in Central and Node, why not to move it into commons and reuse it?